### ISSUE TITLE
stats missing from qpsol

### ISSUE THREAD

**Issue by jgillis** [casadi/casadi#1838]

**Comment by nielsvd** [comment]
bump

**Comment by jgillis** [comment]
bump

**Comment by jgillis** [comment]
bump

**Comment by jgillis** [comment]
(Nasty) workaround now available since we have MX introspection now:

```python
x = SX.sym("x")

nlp = {"x": x, "f": x}

solver = qpsol("solver","clp",nlp)

solver(x0=0)

for i in range(solver.n_instructions()):
  if solver.instruction_id(i)==OP_CALL:
    x = solver.instruction_MX(i)
    f = x.which_function()
    if "qpsol" in f.name():
      print f.stats()
```

**Comment by vdwees** [comment]
Is this fixed now in the latest release? In 3.4.0` it was a problem, but seems be fixed in `3.4.4`.

**Comment by jgillis** [comment]
Yes fixed in https://github.com/casadi/casadi/commit/5a163657e72bb692a7c8710590e4ae8c071c82d1
The 'fix' is in fact the same very ugly hack from above. Real solution will come with #2106
